Product/Production/Services/DocumentSubmissionCore/src/test/java/gov/hhs/fha/nhinc/docsubmission/inbound/StandardInboundDocSubmissionTest.java

```diff
@@ -73,8 +74,8 @@ public void standardInboundDocSubmission() throws PropertyAccessException {
 
         AdapterDocSubmissionProxyObjectFactory adapterFactory = mock(AdapterDocSubmissionProxyObjectFactory.class);
         AdapterDocSubmissionProxy adapterProxy = mock(AdapterDocSubmissionProxy.class);
-        XDRAuditLogger auditLogger = mock(XDRAuditLogger.class);
-
+        DocSubmissionAuditLogger auditLogger = mock(DocSubmissionAuditLogger.class);
+        Properties webContextProperties = new Properties();
```

Comment:
Here I think you can move above initialization of various variables to setUp() method instead of declaring and initializing each time in three test methods